pages/verification/index.tsx

```diff
@@ -48,10 +53,13 @@ export default function Verification(props: PageProps) {
         })
     }
 
-    const sendAnonymousMessage = (anonymousToken: string) => {
+    const sendAnonymousMessage = (anonymousToken: string, optionalParams: SendAnonymousMessageOptionalParams = {}) => {
+        const {replaceRoute = false, withLoading = false} = optionalParams
+
         const member = localStorage.getItem(MessageEnum.targetUser);
         const message = localStorage.getItem(MessageEnum.tempMessage);
         apiAnonymous.defaults.headers.common['Authorization'] = `Bearer ${anonymousToken}`;
+        if(withLoading) setIsLoading(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendAnonymousMessage` function has been modified to include optional parameters. However, there is no null check for the `anonymousToken` parameter. If a null or undefined value is passed, it could lead to potential issues when setting the 'Authorization' header.
```diff
-    const sendAnonymousMessage = (anonymousToken: string, optionalParams: SendAnonymousMessageOptionalParams = {}) => {
+    const sendAnonymousMessage = (anonymousToken: string | null, optionalParams: SendAnonymousMessageOptionalParams = {}) => {
         const {replaceRoute = false, withLoading = false} = optionalParams

         const member = localStorage.getItem(MessageEnum.targetUser);
         const message = localStorage.getItem(MessageEnum.tempMessage);
+        if (!anonymousToken) {
+            console.error('Anonymous token is missing');
+            return;
+        }
         apiAnonymous.defaults.headers.common['Authorization'] = `Bearer ${anonymousToken}`;
         if(withLoading) setIsLoading(true);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->